### DISCUSSION TITLE
French historical sample set

### DISCUSSION THREAD

**Discussion by rousseldenis** [GrandOrgue/grandorgue#1637] [General]
Hi all,

I'm currently looking for a new historical french organ (I already have La Chaise Dieu one - which is very good) and I hesitate between two:

- Dom Bedos - which is quite impressive but recording is quite old (I don't know if this is a criteria).
- Prytanee - I like the sound which is quite clear

If you have any feedback, I will be pleased! Thanks

PS: @larspalo I know you have St Michel en Thierarche, what's your opinion?

**Comment by larspalo** [comment]
It's very difficult to say which you'll like better... I'd say that it depends on your expectations.

Dom Bedos is without a doubt the most versatile of the three if you'd like to play later music too, but it's a modern build. For strictly the "classic" french baroque, and older, I'd say that any of Prytanee and St Michel is really a better choice, and between them it seems that St Michel has much more original pipe material and was less changed over time.

I personally find St Michel to be absolutely gorgeous, and have all the necessary and typical french sounds, but that's perhaps my biased opinion as I've bought it... Prytanee has a few more stops than St Michel and many more sound "perspectives" (channels), but basically fill the same niche stylistically. They are of course slightly different even when comparing roughly the same perspectives and stops, but each organ is a personality of its own and has the right to be so. The benefits of more available perspectives in the Prytanee allows you to select more alternatives (you don't need to use all perspectives at once - especially if you don't have very much RAM in your computer), but it also makes the sample set "heavy" for the computer.

No matter which you choose you'll have another great organ to play! But as I summarized in the second section: Dom Bedos is more versatile than Prytanee, and also much less original. Compare the demos and think about how you'd like to use the organ, that's the best advice I can give.

**Comment by eturpault** [comment]
As said by Palo the best is that you test the two demo versions in order to evaluate how you feel them and which one is rendered the best by your material (audio amplifier and loudspeakers).
I can speak about the Prytanée that I have bought and I play regularly, I confirm that its sound is very clear and it is well recorded. I prefered this sample set over others due to the fact that it has been recorded recently (2021) so I hope the last recording technics have been used to ensure a good sound quality. The specific French stops like Trompette, Tierce, Narard, Cornet are very nice to play French baroque music (Dom Bédos organ should have the same characteristics).
The Pédale division have very few stops which is a drawback to play other kinds of music. The Bourdon 16' of the Grand Orgue division has been linked to the Pédale by Sonus Paradisi which is good but not sufficient. I added in the ODF a coupler to play the Positif stops from the Pédale.
The unequal temperament and the low original pitch of this organ can fortunately be cancelled in GrandOrgue by forcing an equal temperament if needed.
